include/alpaka/platform/PlatformUniformCudaHipRt.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2022 Benjamin Worpitz, René Widera, Andrea Bocci, Bernhard Manfred Gruber, Antonio Di Pilato
+/* Copyright 2022 Benjamin Worpitz, René Widera, Andrea Bocci, Bernhard Manfred Gruber, Antonio Di Pilato, Christian Kaever
  * SPDX-License-Identifier: MPL-2.0
  */
```

Comment:
Your name needs to be in a new line and aligned to `Benjamin Worpitz` above. Otherwise clang-format will complain since you are exceeding the character limit.

Reply:
Thanks for the tip!   clang-fomat should no longer complain, when running
`find -iname '*.cpp' -o -iname '*.hpp' | xargs clang-format-14 -i` (taken from [here](https://github.com/alpaka-group/alpaka/blob/develop/CONTRIBUTING.md)) clang-format changes only files that are in `thirdParty/catch2`.